buildSrc/src/main/java/PosthogBuildConfig.kt

```diff
@@ -14,7 +14,7 @@ object PosthogBuildConfig {
     }
 
     object Android {
-        val COMPILE_SDK = 33
+        val COMPILE_SDK = 34
```

Comment:
```suggestion
        val COMPILE_SDK = 33
```

Since 34 breaks compatibility with older versions, See https://github.com/PostHog/posthog-android/pull/98